<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite start="1553684754900" stop="1553684855363" version="1.5.4" xmlns:ns2="urn:model.allure.qatools.yandex.ru">
    <name>Suite : InvoiceTest</name>
    <title>Suite : InvoiceTest</title>
    <test-cases>
        <test-case start="1553684754889" stop="1553684754895" status="pending">
            <name>verifyDetailsofInvoice</name>
            <failure>
                <message>Test not implemented yet</message>
                <stack-trace>There are no stack trace</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="LT062360"/>
                <label name="thread" value="14960@LT062360.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1553684754896" stop="1553684754896" status="pending">
            <name>ApprovedInvoice</name>
            <failure>
                <message>Test not implemented yet</message>
                <stack-trace>There are no stack trace</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="LT062360"/>
                <label name="thread" value="14960@LT062360.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1553684754897" stop="1553684754897" status="pending">
            <name>RejectInvoice</name>
            <failure>
                <message>Test not implemented yet</message>
                <stack-trace>There are no stack trace</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="LT062360"/>
                <label name="thread" value="14960@LT062360.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1553684754898" stop="1553684754898" status="pending">
            <name>SearchInvoice</name>
            <failure>
                <message>Test not implemented yet</message>
                <stack-trace>There are no stack trace</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="LT062360"/>
                <label name="thread" value="14960@LT062360.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1553684754902" stop="1553684754902" status="pending">
            <name>verifyDetailsofInvoice</name>
            <failure>
                <message>Test not implemented yet</message>
                <stack-trace>There are no stack trace</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="LT062360"/>
                <label name="thread" value="14960@LT062360.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1553684754903" stop="1553684754903" status="pending">
            <name>ApprovedInvoice</name>
            <failure>
                <message>Test not implemented yet</message>
                <stack-trace>There are no stack trace</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="LT062360"/>
                <label name="thread" value="14960@LT062360.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1553684754904" stop="1553684754904" status="pending">
            <name>RejectInvoice</name>
            <failure>
                <message>Test not implemented yet</message>
                <stack-trace>There are no stack trace</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="LT062360"/>
                <label name="thread" value="14960@LT062360.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1553684754905" stop="1553684754905" status="pending">
            <name>SearchInvoice</name>
            <failure>
                <message>Test not implemented yet</message>
                <stack-trace>There are no stack trace</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="LT062360"/>
                <label name="thread" value="14960@LT062360.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1553684754925" stop="1553684838250" status="broken">
            <name>PDFDownload</name>
            <failure>
                <message>NullPointerException: null</message>
                <stack-trace>java.lang.NullPointerException
	at com.sg.Page.InvoiceListPage.PDFDownload(InvoiceListPage.java:242)
	at WAFMTest.InvoiceListTest.PDFDownload(InvoiceListTest.java:36)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Suite"/>
                <label name="testGroup" value="InvoiceTest"/>
                <label name="testClass" value="WAFMTest.InvoiceListTest"/>
                <label name="testMethod" value="PDFDownload"/>
                <label name="host" value="LT062360"/>
                <label name="thread" value="14960@LT062360.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1553684754925" stop="1553684838250" status="broken">
            <name>PDFDownload</name>
            <failure>
                <message>NullPointerException: null</message>
                <stack-trace>java.lang.NullPointerException
	at com.sg.Page.InvoiceListPage.PDFDownload(InvoiceListPage.java:242)
	at WAFMTest.InvoiceListTest.PDFDownload(InvoiceListTest.java:36)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Suite"/>
                <label name="testGroup" value="InvoiceTest"/>
                <label name="testClass" value="WAFMTest.InvoiceListTest"/>
                <label name="testMethod" value="PDFDownload"/>
                <label name="host" value="LT062360"/>
                <label name="thread" value="14960@LT062360.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1553684840469" stop="1553684852976" status="broken">
            <name>AddApprover</name>
            <failure>
                <message>IndexOutOfBoundsException: Index: 1, Size: 0</message>
                <stack-trace>java.lang.IndexOutOfBoundsException: Index: 1, Size: 0
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at com.sg.Page.InvoiceListPage.AddApprover(InvoiceListPage.java:253)
	at WAFMTest.InvoiceListTest.AddApprover(InvoiceListTest.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Suite"/>
                <label name="testGroup" value="InvoiceTest"/>
                <label name="testClass" value="WAFMTest.InvoiceListTest"/>
                <label name="testMethod" value="AddApprover"/>
                <label name="host" value="LT062360"/>
                <label name="thread" value="14960@LT062360.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1553684840469" stop="1553684852976" status="broken">
            <name>AddApprover</name>
            <failure>
                <message>IndexOutOfBoundsException: Index: 1, Size: 0</message>
                <stack-trace>java.lang.IndexOutOfBoundsException: Index: 1, Size: 0
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at com.sg.Page.InvoiceListPage.AddApprover(InvoiceListPage.java:253)
	at WAFMTest.InvoiceListTest.AddApprover(InvoiceListTest.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Suite"/>
                <label name="testGroup" value="InvoiceTest"/>
                <label name="testClass" value="WAFMTest.InvoiceListTest"/>
                <label name="testMethod" value="AddApprover"/>
                <label name="host" value="LT062360"/>
                <label name="thread" value="14960@LT062360.main(1)"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TestNG"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
